src/components/Envelope.vue

```diff
@@ -455,9 +456,8 @@ list-item-style.draft .app-content-list-item-line-two {
 }
 
 .icon-reply {
-	background-image: url('../../img/reply.svg');
```

Comment:
where's the icon loaded from if this is dropped? :thinking:

Reply:
web console says its from mail.css